index.cjs

```diff
@@ -70,7 +71,7 @@ module.exports = function config(options = {}) {
     },
     {
       extends: [
-        reactPlugin.configs.flat.recommended,
+        reactPlugin.configs.flat["recommended"],
```

Comment:
`// @ts-check` reporting wanting the same format as the `jsx-runtime` check below, due to string indexing.